scripts/exgdas_global_marine_analysis_vrfy_manual.py

```diff
@@ -0,0 +1,229 @@
+#!/usr/bin/env python3
+################################################################################
+#  UNIX Script Documentation Block
+#                      .                                             .
+# Script name:         exufsda_global_marine_analysis_vrfy.py
+# Script description:  State and observation space verification for the
+#                      UFS Global Marine Analysis
+#
+# Author: Guillaume Vernieres      Org: NCEP/EMC     Date: 2023-01-23
+#
+# Abstract: This script produces figures relevant to the marine DA cycle
+#
+# $Id$
+#
+# Attributes:
+#   Language: Python3
+#
+################################################################################
```

Comment:
you can remove the header, this is not going to be part of the workflow

Reply:
Sure-